ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -492,6 +498,59 @@ const DiskSizeSelector = ({onChange, disabled, selectedDiskSize, diskSize, idPre
     </FlexRow>;
 };
 
+const GpuConfigSelector = ({disabled, onChange, selectedMachine, gpuConfig})  => {
+  const {
+    gpuType = 'nvidia-tesla-t4',
+    numOfGpus = 1,
+  } = gpuConfig || {};
+  const [selectedGpuType, setSelectedGpuType] = useState<string>(gpuType);
+  const [selectedNumOfGpus, setSelectedNumOfGpus] = useState<number>(numOfGpus);
+  const validGpuOptions = getValidGpuTypes(selectedMachine.cpu, selectedMachine.memory);
+  const validGpuNames = fp.flow(fp.map('name'), fp.uniq, fp.sortBy('price'))(validGpuOptions);
+  const validNumGpusOptions = fp.flow(fp.filter({ type: selectedGpuType }), fp.map('numGpus'))(validGpuOptions);
+  const [enableGpu, setEnableGpu] = useState<boolean>(!!gpuConfig);
+
+  useEffect(() => {
+    onChange(enableGpu && validGpuOptions.length > 0 ? {
+      gpuType: selectedGpuType,
+      numOfGpus: selectedNumOfGpus,
+    } : null);
+  }, [enableGpu, selectedGpuType, selectedNumOfGpus]);
+
+  return <fieldset style={{marginTop: '0.75rem'}}>
+    <div>
+      <div>
+        <input name='enable-gpu'
+               type='checkbox' checked={enableGpu}
+               onChange={() => {
+                 setEnableGpu(!enableGpu);
+               }}/>Enable GPUs
```

Comment:
This can be a follow-up, but I think having a support link next to this will be very helpful